services/app/src/components/views/Search.test.tsx

```diff
@@ -131,7 +126,7 @@ describe('Search', () => {
   });
 
   describe('Details', () => {
-    function setup(questTitle: string, overrides?: Partial<SearchDetailsProps>) {
+    function setup(questTitle: string, overrides?: Partial<SearchDetailsProps>, questOverrides?: Partial<QuestDetails>) {
```

Comment:
Ah, it's because you aren't actually using the questOverrides parameter below, you need to merge it into props.quest like the other setup function does - https://github.com/ExpeditionRPG/expedition/pull/6/files/c290f04aa0711f9a4d53d1a1bf9ea9a6d047cc3f..2cfb3abeb518c21e4e8cee10bb5dcad6dcb2468a#diff-c2b51ae0add67354cb1c426c9d14a3bcR69

Reply:
Fixed and pushed.  Thanks!